### ISSUE TITLE
SearchReportsWithHttpInfo vs SearchReportsAsyncWithHttpInfo incomplete code within SearchReportsWithHttpInfo

### ISSUE THREAD

**Issue by xantari** [CyberSource/cybersource-rest-client-dotnet#44]
In SearchReportsAsyncWithHttpInfo you have the following code:

```
// to determine the Content-Type header
String[] localVarHttpContentTypes = new String[] {
  "application/json;charset=utf-8"
};
String localVarHttpContentType = Configuration.ApiClient.SelectHeaderContentType(localVarHttpContentTypes);
```

But in the non-async version (SearchReportsWithHttpInfo ) it seems it was forgot to be set:

```
// to determine the Content-Type header  
String[] localVarHttpContentTypes = new String[] {  
   "application/json;charset=utf-8"  
};  
String localVarHttpContentType = "";  
```

Is this stuff not getting code reviewed?

Where are the unit tests for this stuff?

**Comment by xantari** [comment]
Cybersource now tracking this issue under cybsdx-104 internal bug identifier

**Comment by xantari** [comment]
It appears this was fixed in the 0.0.0.10 release...

**Comment by snavinch** [comment]
Closing this as it has been fixed in the latest version.
